### PR TITLE
Containers: add conditions for Leap Micro 

### PR DESCRIPTION
**Pull request by jlausuch**
[MicroOS](https://openqa.opensuse.org/tests/2283217)
[Leap Micro](https://openqa.opensuse.org/tests/2283218)  (error should be fixed by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14648)
[SLE Micro](https://openqa.suse.de/tests/8470872)
[SLE 12-SP5](http://openqa.suse.de/t8470873)
[SLE 15-SP1](http://openqa.suse.de/t8470874)

### CODE REVIEW

**Review by pdostal — APPROVED** [review]

**Comment by jlausuch** [comment]
> I'm against this change: Functional disadvantages by implicitly assuming that we will never run container test on transactional system for aesthetic reasons only, is not something that I do not find a good practice. Our future selves might be unhappy ....

Ok, you convinced me :)

File: lib/main_containers.pm [hunk 1]
```diff
@@ -77,7 +77,7 @@ sub load_image_tests_docker {
 
 sub load_host_tests_podman {
     my ($run_args) = @_;
-    if (is_leap('15.1+') || is_tumbleweed || is_sle("15-sp1+") || is_sle_micro || is_microos) {
+    if (is_leap('15.1+') || is_tumbleweed || is_sle("15-sp1+") || is_sle_micro || is_microos || is_leap_micro) {
```
Comment (mloviska) [inline lib/main_containers.pm:80]: Hm...Doesn't it already mean run everywhere except of sle <= 15 ?

Comment (jlausuch) [reply lib/main_containers.pm]: done, and new VRs

**Review by grisu48 — APPROVED** [review]
LGTM!

**Pull request merged**